### PR TITLE
use asterisk for dependency cycle too

### PR DESCRIPTION
**Pull request by ChannyClaus**
## Summary
https://github.com/astral-sh/uv/pull/4611#issuecomment-2196159445 https://github.com/astral-sh/uv/pull/4449#issuecomment-2184174557 this change was lost in merge conflict resolution :.(

## Test Plan
almost purely aesthetic change 🌵

### CODE REVIEW

**Comment by ChannyClaus** [comment]
@zanieb fyi

File: crates/uv/tests/pip_tree.rs [hunk 1]
```diff
@@ -623,8 +623,8 @@ fn cyclic_dependency() {
     uv-cyclic-dependencies-c v0.1.0
     └── uv-cyclic-dependencies-a v0.1.0
         └── uv-cyclic-dependencies-b v0.1.0
-            └── uv-cyclic-dependencies-a v0.1.0 (#)
-    (#) Dependency cycle
+            └── uv-cyclic-dependencies-a v0.1.0 (*)
+    (*) Package tree already displayed
```
Comment (ibraheemdev) [inline crates/uv/tests/pip_tree.rs:627]: Why is the message changing?

**Review by ibraheemdev — APPROVED** [review]

File: crates/uv/src/commands/pip/tree.rs [hunk 2]
```diff
@@ -57,14 +57,12 @@ pub(crate) fn pip_tree(
     .join("\n");
     writeln!(printer.stdout(), "{rendered_tree}").unwrap();
     if rendered_tree.contains('*') {
-        writeln!(
-            printer.stdout(),
-            "{}",
+        let message = if no_dedupe {
+            "(*) Dependency cycle".italic()
```
Comment (zanieb) [inline crates/uv/src/commands/pip/tree.rs:61]: Should we say "Package tree is a cycle and cannot be shown" as we discussed before?

Comment (ibraheemdev) [reply crates/uv/src/commands/pip/tree.rs:61]: Or "Package tree forms/contains a cycle and cannot be shown"

**Pull request merged**